spec/win32/unit/certstore_spec.rb

```diff
@@ -119,7 +119,7 @@
         allow(certstore).to receive(:open).with(store_name).and_return(certstore_obj)
         allow(certstore_obj).to receive(:read_certificate_content).with(cert_file_path).and_return("")
         store = certstore.open(store_name)
-        expect { store.add(cert_file_path) }.to raise_error("The operation was canceled by the user.")
+        expect { store.add(cert_file_path) }.to raise_error(SystemCallError)
```

Comment:
It is nice to see the actual error that would be raised.

Reply:
Right @PrajaktaPurohit  we were thinking for having the exception handling task in other card and so for now we modified the spec to check the error class rather than standard error message.  Currently the SystemCallError is  raising different error for e.g If I am expecting "ASN1 unexpected end of data" it throws "Cannot fine object or property"

Reply:
for sure!